src/plugins/kibana_react/public/overview_page/overview_page_footer/overview_page_footer.tsx

```diff
@@ -104,26 +104,6 @@ export const OverviewPageFooter: FC<Props> = ({
         <EuiFlexItem grow={false}>
           <div>{isAdvancedSettingsEnabled ? defaultRoutebutton : null}</div>
```

Comment:
This looks great, thanks for making the changes!

Now that we only have the one conditional link in the footer, should we wrap the entire footer in this conditional check? In other words, return `null` / no `<footer...` if `isAdvancedSettingsEnabled` is false? Otherwise, we're just rendering an empty footer section.

Reply:
Done!